docs/INSTALL.md

```diff
@@ -0,0 +1,106 @@
+# Environment Setup to Use IRL-1
+
+## 0. Install Ubuntu 16.04 LTS
+Download ISO File: https://www.ubuntu.com/download/desktop
+
+## 1. Install ROS Kinetic & Gazebo
+From: http://wiki.ros.org/kinetic/Installation/Ubuntu
+
+### 1.1. Add ROS repository to apt
+```bash
+sudo sh -c 'echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list'
+sudo apt-key adv --keyserver hkp://ha.pool.sks-keyservers.net:80 --recv-key 421C365BD9FF1F717815A3895523BAEEB01FA116
+sudo apt update
+```
+
+### 1.2. Install ROS
+Includes: ROS, Gazebo, rqt, rviz, robot-generic libraries, 2D/3D simulators, navigation and 2D/3D perception
+
+Download: 495 MB
+Install: 2497 MB
+
+```bash
+sudo apt-get install ros-kinetic-desktop-full
```

Comment:
Ideally, you don't install the full desktop environment on the robot itself, but I have to check which distribution is the best to get all dependencies, so let's leave it like this for now.